1.Core/UIElements/Drivers/GenericWebDriver.cs

```diff
@@ -25,8 +25,14 @@ public static void Dispose()
     public static IWebDriver LoadDriver()
     {
         var key = ConfigBuilder.Instance.GetString("ui", "DriverLocation");
-        return key.Equals("Local")
-            ? WebDriverFactory.GetDriver(ConfigModel.DriverType)
-            : RemoteWebDriverFactory.GetDriver(ConfigModel.DriverType);
+        switch (key)
+        {
+            case "BrowserStack":
+                return BrowserstackWebDriverFactory.GetDriver();
+            case "Local":
+                return WebDriverFactory.GetDriver(ConfigModel.DriverType);
+            default:
+                return RemoteWebDriverFactory.GetDriver(ConfigModel.DriverType);
```

Comment:
There should not be a default result on this switch statement, you could create another case for "Remote" so we can run tests on a custom Remote Selenium Hub driver.